HoloPipelines/models/dense_vnet_abdominal_ct/config.ini

```diff
@@ -1,12 +1,10 @@
-############################ input configuration sections
```

Comment:
Why did you delete these comments? They are actually kinda helpful, especially when you start manipulating this file